hack/e2e.sh

```diff
@@ -2,15 +2,11 @@
 
 set -euo pipefail
 
-# Path to a clouds.yaml to use for e2e tests.
-E2E_OSCLOUDS=${E2E_OSCLOUDS:-/etc/openstack/clouds.yaml}
+# Move to top level directory
+REAL_PATH=$(realpath $0)
+cd "$(dirname "$REAL_PATH")/.."
```

Comment:
Will this work?
```suggestion
REAL_PATH=$(git rev-parse --show-toplevel)
cd $REAL_PATH
```

Reply:
Please change the variable name too, call it TOP_LEVEL or something alike.

Reply:
or drop the variable entirely if we don't need it.